tools/sdk/lib/fix_sdk_libs.sh

```diff
@@ -98,3 +166,12 @@ if [[ $(sha256sum user_interface.o | awk '{print $1}') != $uics || $(sha256sum e
 	xtensa-lx106-elf-ar r libmain.a eagle_lwip_if.o user_interface.o
 fi
 rm -f eagle_lwip_if.o user_interface.o eap.o
+
+if [[ ${VERSION} == "NONOSDK3V0" ]]; then
+	xtensa-lx106-elf-objcopy --weaken-symbol load_non_32_wide_handler libmain.a
+elif [[ ${VERSION:0:9} == "NONOSDK30" ]]; then
```

Comment:
nit: for this and the above, bash has a pretty good string matcher through `case`
```bash
case $VERSION in
"NONOSDK3V0") echo "foo" ;;
"NONOSDK30"*) echo "bar" ;;
"I write this in Github Web, syntax may be wrong") echo "baz" ;;
esac
```